src/global.h

```diff
@@ -362,15 +362,214 @@ class CCustomEvent : public QEvent
+        if ( currentIndex < appArgc )
+        {
+            currentArgv++;
+            currentIndex++;
+
+            if ( currentIndex < appArgc )
+            {
+                return QString ( *currentArgv );
+            }
+        }
+
+        return QString();
+    }
+};
+
+// defines for commandline options in the style "shortopt", "longopt"
+// Name is standard CMDLN_LONGOPTNAME
+// These defines can be used for strShortOpt, strLongOpt parameters
+// of the CCommandlineOptions functions.
+//
+// clang-format off
+#define CMDLN_SERVER              "-s",                    "--server"
+#define CMDLN_INIFILE             "-i",                    "--inifile"
+#define CMDLN_NOGUI               "-n",                    "--nogui"
+#define CMDLN_PORT                "-p",                    "--port"
+#define CMDLN_QOS                 "-Q",                    "--qos"
+#define CMDLN_NOTRANSLATION       "-t",                    "--notranslation"
+#define CMDLN_ENABLEIPV6          "-6",                    "--enableipv6"
+#define CMDLN_DISCONONQUIT        "-d",                    "--discononquit"
+#define CMDLN_DIRECTORYSERVER     "-e",                    "--directoryserver"
+#define CMDLN_DIRECTORYFILE       "--directoryfile",       "--directoryfile"
+#define CMDLN_LISTFILTER          "-f",                    "--listfilter"
+#define CMDLN_FASTUPDATE          "-F",                    "--fastupdate"
+#define CMDLN_LOG                 "-l",                    "--log"
+#define CMDLN_LICENCE             "-L",                    "--licence"
+#define CMDLN_HTMLSTATUS          "-m",                    "--htmlstatus"
+#define CMDLN_SERVERINFO          "-o",                    "--serverinfo"
+#define CMDLN_SERVERPUBLICIP      "--serverpublicip",      "--serverpublicip"
+#define CMDLN_DELAYPAN            "-P",                    "--delaypan"
+#define CMDLN_RECORDING           "-R",                    "--recording"
+#define CMDLN_NORECORD            "--norecord",            "--norecord"
+#define CMDLN_SERVERBINDIP        "--serverbindip",        "--serverbindip"
+#define CMDLN_MULTITHREADING      "-T",                    "--multithreading"
+#define CMDLN_NUMCHANNELS         "-u",                    "--numchannels"
+#define CMDLN_WELCOMEMESSAGE      "-w",                    "--welcomemessage"
+#define CMDLN_STARTMINIMIZED      "-z",                    "--startminimized"
+#define CMDLN_CONNECT             "-c",                    "--connect"
+#define CMDLN_NOJACKCONNECT       "-j",                    "--nojackconnect"
+#define CMDLN_MUTESTREAM          "-M",                    "--mutestream"
+#define CMDLN_MUTEMYOWN           "--mutemyown",           "--mutemyown"
+#define CMDLN_CLIENTNAME          "--clientname",          "--clientname"
+#define CMDLN_CTRLMIDICH          "--ctrlmidich",          "--ctrlmidich"
+// Backwards compatibilyty:
+#define CMDLN_CENTRALSERVER       "--centralserver",       "--centralserver"
+// pgScorpio: TODO These are NOT in help !:
```

Comment:
I remember that they are mostly used for debugging/advanced use. Hence it was agreed not to document them.

Reply:
@ann0see 

But these options are documented on the [wiki](https://jamulus.io/wiki/Server-Troubleshooting)  and the [man page](https://manpages.debian.org/testing/jamulus/Jamulus.1.en.html)

So should I change this comment to "Debug options: (not in help)" ?

Also I think I'm messing things up, since this is a change from issue #2538. 
And I only just now realize what you ment with "although merging from your master branch to our master branch is discouraged". I didn't branch this one ! So now, after my rebasing these changes also appeared in my branches i guess?
(Sorry, still learning...)

Reply:
If "show all servers" and "analszer console" are really documented somewhere, something I wasn’t aware of changed (@gilgongo do you know anything?).


If you rebase your master branch, sure this will show up in the PR from master (but shouldn’t on this one).

My offer still stands: we can have a short call if you need it).

Reply:
Yes, the **use** of "show all servers" is mentioned as a way of debugging a problem. But "analyzerconsole" got added to the man page, because... the man page plays by its own rules.